whois/Cargo.toml

```diff
@@ -17,3 +17,12 @@ polkadot-introspector-essentials = { workspace = true }
 polkadot-introspector-priority-channel = { workspace = true }
 thiserror = { workspace = true }
 tokio = { workspace = true }
+serde = {workspace = true }
+serde_json = {workspace = true }
+itertools = "0.13"
+serde-binary = "0.5"
+hex = "0.4.3"
+subp2p-explorer-cli = { git = "https://github.com/alexggh/subp2p-explorer", branch="alexggh/make-subp2p-usable-as-library"}
+subp2p-explorer = { git = "https://github.com/alexggh/subp2p-explorer", branch="alexggh/make-subp2p-usable-as-library"}
+
+libp2p = { version = "0.52.0", features = ["dns", "identify", "kad", "macros", "mdns", "noise", "ping", "tcp",  "tokio", "yamux", "websocket", "request-response"] }
```

Comment:
last is 0.54.1. is it set deliberatly?

Reply:
It should be whatever version we currently have in polkadot.